database/initial_structure.sql

```diff
@@ -0,0 +1,37 @@
+CREATE SEQUENCE batch_id_seq
+    INCREMENT 1
+    START 1;
+
+CREATE SEQUENCE interaction_id_seq
```

Comment:
I'm guessing `batch_id_seq` will be used for `batch_id`, but what about `interaction_id_seq`?